src/app/cli/tests/tests/test_datafusion_cli.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+use assert_cmd::Command;
+
+#[test]
+fn test_datafusion_cli() {
```

Comment:
please add `#[test_group::group(engine, datafusion)]` as per [guidelines](https://github.com/kamu-data/kamu-cli/blob/master/DEVELOPER.md#test-groups)

Reply:
thank you, done